Files/UserControls/FilePreviews/TextPreview.xaml.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Files.Filesystem;
+using Files.ViewModels;
+using Files.ViewModels.Properties;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Storage;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Documents;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace Files.UserControls.FilePreviews
+{
+    public sealed partial class TextPreview : PreviewControlBase
```

Comment:
MVVM 😶

Reply:
@d2dyno1 Can you tell me specifically what you want me to change?

Reply:
We want to merge into MVVM pattern sometime in the future. It'd be better if new PRs came with MVVM

Reply:
@d2dyno1 What exactly do I need to change?

Reply:
Ideally you'd have to convert all `.xaml.cs` (code-behind) files to View Models (`somefileViewModel`). If you have any problems with it, check out #2881, it follows MVVM pattern and it can help answer some of your questions :)